polling_stations/apps/file_uploads/utils.py

```diff
@@ -12,17 +12,58 @@ def get_domain(request):
     return os.environ.get("APP_DOMAIN", request.META.get("HTTP_HOST"))
 
 
+def approved_custom_domain(user, email_domain):
+    # This is a quick fix: Peter can raise a
+    # PR to update this list with the the actual
+    # list of email domains and councils as the
+    # need arises.
+
+    custom_domains = {
+        "custom_council.gov.uk": "Foo Council",
+        "custom_council2.gov.uk": "Bar Council",
+    }
```

Comment:
Note: Use the `council.gss` with 3 letter code values rather than the `council.name`